lib/banner.dart

```diff
@@ -138,8 +149,9 @@ class _DFPBannerViewController {
   final void Function() onAdOpened;
   final void Function() onAdClosed;
   final void Function() onAdLeftApplication;
+  final void Function(DFPBannerViewController controller) onAdViewCreated;
 
-  _DFPBannerViewController({
+  DFPBannerViewController({
```

Comment:
I want the constructor to be private because of not to let they make an instance by mistake.

```
DFPBannerViewController._internal
```

Reply:
Done